app/models/queued_email/add_record_to_herbarium.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Add HerbariumRecord Not Curator Email
+class QueuedEmail
+  class AddRecordToHerbarium < QueuedEmail
+    def herbarium_record
+      get_object(:herbarium_record, ::HerbariumRecord)
+    end
+
+    def self.create_email(sender, recipient, herbarium_record)
+      result = create(sender, recipient)
+      raise("Missing herbarium_record!") unless herbarium_record
+
+      result.add_integer(:herbarium_record, herbarium_record.id)
+      result.finish
+      result
+    end
+
+    def deliver_email
+      # Make sure it hasn't been deleted since email was queued.
+      return unless herbarium_record
+
+      AddHerbariumRecordMailer.build(user, to_user,
+                                     herbarium_record).deliver_now
+    end
+  end
+end
```

Comment:
@pellaea, @JoeCohen 

I think Joe had originally sketched out this `QueuedEmail` class, but in the code thus far, had not implemented it, and it had no test yet in `QueuedEmailTest`. 

I added the `reply_to` in the `AddHerbariumRecordMailer`, because it seemed from the way `HerbariaControllerTest` had been written that this should be set by the mailer. It wasn't set before, though, and it was passing tests.

Reply:
Half-finished code sounds more like my style, honesty, but thanks anyway for fixing it!  I will trust that you've got it down now and that your added reply-to is appropriate.

Reply:
update... `reply_to` was not appropriate! Reverted.